src/components/SitewideFooter.js

```diff
@@ -32,14 +32,21 @@ const useStyles = createUseStyles((theme) => ({
 
 const SitewideFooter = () => {
   const classes = useStyles();
+  const data = useStaticQuery(query);
+  const { newsletterUrl } = data.site.siteMetadata;
 
   return (
     <footer>
       <div className={classes.inner}>
         <div>© {new Date().getFullYear()} Larder, Inc. All rights reserved.</div>
 
         <nav>
-          <Link to="/terms">Terms of service</Link>
+          <a href={newsletterUrl} target="_blank" rel="noopener noreferrer">
+            Backstage Weekly
+          </a>
+          <Link to="/terms" className={classes.nonFirstLink}>
```

Comment:
out of curiosity what is  `classes.nonFirstLink`?

Reply:
I'm adding spacing between the links to keep them apart from each other but the spacing is not required on the first link in the list.